packages/client/e2e/analytics.spec.ts

```diff
@@ -0,0 +1,145 @@
+
+test.describe("Analytics", () => {
+  let analyticsPage: Page
+
+  test.beforeAll(async ({ context, fxPagesRec }, workerInfo) => {
+    if (isOpenFin(workerInfo)) {
+      analyticsPage = fxPagesRec["fx-analytics"]
+    } else {
+      const pages = context.pages()
+      const mainWindow = pages.length > 0 ? pages[0] : await context.newPage()
+
+      await mainWindow.goto(`${process.env.E2E_RTC_WEB_ROOT_URL}`)
+
+      analyticsPage = mainWindow
+    }
+  })
+
+  test.describe("Profit & Loss section", () => {
+    test("Last position amount is displayed in numerical format @smoke", async () => {
+      const lastPositionAmount = await analyticsPage
+        .locator("[data-testid='lastPosition']")
+        .textContent()
+
+      const regexp = RegExp(`[-+,.0-9]`, "g")
+      expect(lastPositionAmount).toMatch(regexp)
+    })
+
+    test("Last position amount is updated periodically", async () => {
+      const lastpositionLocator = analyticsPage.locator(
+        "[data-testid='lastPosition']",
+      )
+      const initiallastPositionAmount = await lastpositionLocator.textContent()
+
+      if (initiallastPositionAmount)
+        await expect(lastpositionLocator).not.toContainText(
+          initiallastPositionAmount,
+          { timeout: ElementTimeout.NORMAL },
+        )
+    })
+
+    test("Correct text color is displayed based on position value", async () => {
+      const initialpositionLocator = analyticsPage.locator(
+        "[data-testid='lastPosition']",
+      )
+      const amount = Number(
+        (await initialpositionLocator.textContent())?.replace(/[.,]/g, ""),
+      )
+
+      if (amount < 0) {
+        expect(await initialpositionLocator.getAttribute("color")).toEqual(
+          "negative",
+        )
+      } else
+        expect(await initialpositionLocator.getAttribute("color")).toEqual(
+          "positive",
+        )
+    })
+
+    test.skip("Graph display coherent values", async () => {
+      // TODO
+      // more complex. may requires datatest-id to reliably assert graph content
+    })
+  })
+
+  test.describe("Positions section", () => {
+    test("Position nodes are showing tooltip information for each currencies", async () => {
+      for (const currency of currencies) {
+        const currencyCircle = analyticsPage
+          .locator("g.node")
+          .filter({ hasText: currency })
+          .first()
+        const currencyTooltip = analyticsPage
+          .locator("[data-testid='tooltip']", { hasText: currency })
+          .first()
+
+        await currencyCircle.hover()
+        await expect(currencyTooltip).toBeVisible({
+          timeout: ElementTimeout.AGGRESSIVE,
+        })
+        const regexp = RegExp(`${currency} [-+,.0-9]`, "g")
+        expect(
+          await currencyTooltip.textContent(),
+          `tooltip for ${currency} doesn't match expected pattern`,
+        ).toMatch(regexp)
+      }
+    })
+
+    test("Position nodes can be moved", async () => {
```

Comment:
cool - forgot about this (think it just comes with the graph tech) .. fairly useless, tho, innit 😆